src/kindlemint/agents/kdp_performance_agent.py

```
@@ -0,0 +1,652 @@
+        rating_selectors = [
+            '[data-hook="average-star-rating"] .a-offscreen',
+            ".reviewCountTextLinkedHistogram .a-offscreen",
+        ]
+
+        for selector in rating_selectors:
+            element = soup.select_one(selector)
+            if element:
+                text = element.get_text(strip=True)
+                try:
+                    return float(text.split()[0])
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)